librocksdb_sys/rocksdb/db/compaction_picker.cc

```diff
@@ -677,7 +677,7 @@ Compaction* CompactionPicker::CompactRange(
       mutable_cf_options.max_compaction_bytes, output_path_id,
       GetCompressionType(ioptions_, vstorage, mutable_cf_options, output_level,
                          vstorage->base_level()),
-      /* max_subcompactions */ 0, std::move(grandparents),
+      max_subcompactions, std::move(grandparents),
```

Comment:
Does RocksDB master branch has this feature?

Reply:
https://github.com/facebook/rocksdb/pull/4082

Reply:
This is fixed in RocksDB, PTAL.